.travis.yml

```diff
@@ -8,10 +8,10 @@ install:
   - "pip install -r requirements.txt"
   - "pip install -r tests/requirements.txt"
   - "pip install pytest-cov coveralls"
+  - "sudo pip install git+https://github.com/TomasTomecek/dockpulp"
```

Comment:
This is why I added `sudo`: https://travis-ci.org/DBuildService/atomic-reactor/jobs/69115173